src/components/calculator.js

```diff
@@ -0,0 +1,42 @@
+const calculator = () => (
+  <div className="App container">
+    <div className="app">
```

Comment:
The `Calculator.js` component should be class-based as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/178592870-5a16beff-f9cd-495f-a173-43c18f2ef3be.png)

- Kindly refactor your component to use classes.
For example:
```
import React, { Component } from 'react';

class Calculator extends Component {
  render() {
      ....
    };
  }
}

export default Calculator;
```